cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -33,6 +33,8 @@ spec:
         volumeMounts:
           - name: config
             mountPath: /etc/config
+      nodeSelector:
+        beta.kubernetes.io/masq-agent-ds-ready: "true"            
```

Comment:
I think the confusion on my end is that I thought pr/#46473 was reverted.  Which would have made this moot, but it turns out it wasn't.

pr/#47794 should fix the issue.